run.py

```diff
@@ -91,13 +102,15 @@ def get_cmdline_arguments() -> argparse.Namespace:
         dest="doc",
         help="Create documentation for project using Sphinx",
     )
-
     args = parser.parse_args()
-
     return args
 
 
-def perform_tests(test_path: str, test_postfix: str, *test_flags: str) -> str:
+def perform_tests(
+    test_path: str,
+    test_postfix: str,
+    test_flags: Optional[Union[str, List[str]]] = None,
```

Comment:
I am not sure if string here ` test_flags: Optional[Union[str, List[str]]] = None,` will work as expected, further in code `extend` is used 
`if test_flags:
            command.extend(test_flags)`
so when we provide as argument string like "-v" than it will be devided into two command list elements "-" and "v", is it expected? I do not think so.

Reply:
You are right, I will remove str and left only List[str]